src/main/resources/templates/geoweaver.html

```diff
@@ -364,7 +364,7 @@
 						
 					</div>
 
-					<div id="right-page-div" class="row" style="height:calc( 100% - 50px ); left:0; margin:0; padding: 0;padding-bottom:0px;">
+					<div id="right-page-div" class="row" style="height:100%; left:0; margin:0; padding: 0;padding-bottom:0px;">
```

Comment:
Cannot scroll to the bottom. The -50px is to ensure all the page content can be seen.